Sources/TensorFlow/Core/Threading.swift

```diff
@@ -146,6 +146,51 @@ class Thread {
     }
 }
 
+class Mutex {
```

Comment:
Could you please add a simple documentation comment? Even though this is an internal API.

Reply:
Sure!